src/CatalogueInfoPage.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Box } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { ReactComponent as Img1 } from "./img/img1.svg";
+import { ReactComponent as Img2 } from "./img/img2.svg";
+import { ReactComponent as Img3 } from "./img/img3.svg";
```

Comment:
I think we should rename these because there's no way to know what they are intuitively.